Sources/Plasma/PubUtilLib/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-add_definitions(-D_LIB)
+include_directories("${PLASMA_SOURCE_ROOT}/CoreLib")
+include_directories("${PLASMA_SOURCE_ROOT}/FeatureLib")
```

Comment:
Likewise, PubUtilLib shouldn't depend on FeatureLib, but there is a **lot** of reaching across the boundary here for stuff like pfKIMsg and pfCamera stuff :(